pkgs/development/ocaml-modules/lem/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv
+, fetchFromGitHub
+, lib
+, ncurses
+, makeWrapper
+
+, ocamlbuild
+, findlib
+, ocaml
+, num
+, zarith
+}:
+
+stdenv.mkDerivation rec {
+  pname = "lem";
+  version = "2022-12-10";
+
+  minimalOCamlVersion = "4.07";
+
+  nativeBuildInputs = [ makeWrapper ocamlbuild findlib ocaml ];
+  buildInputs = [ ncurses num ];
+  propagatedBuildInputs = [ zarith ];
+
+  installFlags = "INSTALL_DIR=$(out)";
+
+  preInstall = "mkdir -p $OCAMLFIND_DESTDIR";
+  postInstall = ''
+    mv $out/bin/lem $out/bin/.lem_wrapped
+    makeWrapper $out/bin/.lem_wrapped $out/bin/lem --set LEMLIB $out/share/lem/library
```

Comment:
Use `wrapProgram` in stead of `makeWrapper`, which automatically handles creating `.foo-wrapped`.